.github/labeler.yml

```
@@ -113,6 +113,13 @@
         - pkgs/applications/editors/jupyter-kernels/**/*
         - pkgs/applications/editors/jupyter/**/*
 
+"6.topic: k3s":
+  - any:
+    - changed-files:
+      - any-glob-to-any-file:
+        - pkgs/applications/networking/cluster/k3s/**/*
+        - nixos/tests/k3s/**/*
```

```suggestion
        - nixos/tests/k3s/**/*
        - nixos/modules/services/cluster/k3s/**/*
```

Well idk what `/**/*` means, but remember to include this directory

---

Thanks. Totally missed that.

---

I believe `/**/` means to include sub-directories.